srcpkgs/ibus-bamboo/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'ibus-bamboo'
+pkgname=ibus-bamboo
+version=0.4.6
+revision=1
+build_style=gnu-makefile
+short_desc="Vietnamese input method for ibus."
+maintainer="luongthanhlam"
+license="GPL-3"
```

Comment:
need SPDX identifier

Reply:
Thanks for your reply, can i get the SPDX from [here](https://spdx.org/licenses/)?

Reply:
Yes, should be GPL-3.0-or-later